docs/getting-started/benchmark.md

```diff
@@ -0,0 +1,211 @@
+
+```
+============================================================================================================================
+Type         Ops/sec     Hits/sec   Misses/sec    Avg. Latency     p50 Latency     p99 Latency   p99.9 Latency       KB/sec
+----------------------------------------------------------------------------------------------------------------------------
+Gets      4109802.84   4109802.84         0.00         0.40126         0.38300         0.67900         0.90300    296551.68
+```
+
+We can observe that both `Ops` and `Hits` are the same, meaning all of the GET requests
+coming from the load test hit the existing keys.
+Dragonfly responded with returning values for each request and its average QPS was 4.1M qps,
+with P99.9 latency - 903us.
+
+### Read test with pipelining
+
+Here's another way to loadtest Dragonfly. This one is sending SETs with pipeline of batch size 10:
+`memtier_benchmark -s $SERVER_PRIVATE_IP --ratio 0:1 -t 60 -c 5  -n 200000  --distinct-client-seed --hide-histogram --pipeline=10`
+
+```
+ALL STATS
+============================================================================================================================
+Type         Ops/sec     Hits/sec   Misses/sec    Avg. Latency     p50 Latency     p99 Latency   p99.9 Latency       KB/sec
+----------------------------------------------------------------------------------------------------------------------------
+Gets      7083583.57   7083583.57         0.00         0.45821         0.44700         0.69500         1.53500    511131.14
+```
+
+During the pipelining mode, memtier_benchmark sends `K` requests in batch without before waiting
+for them to complete. In this case, `K` is `10`. Pipelining reduces the CPU load spent
+in the networking stack, and as a result, Dragonfly can reach 7M qps with sub-millisecond latency.
+Please note, that for real world usecases, pipelining requires cooperation of a client side app,
+which must send multiple requests on a single connection before waiting for the server to respond.
+
+Some asynchronous client libraries like `StackExchange.Redis` or `ioredis` allow multiplexing requests
+on a single connection. They can still provide a simplified synchronous interface to their users while
+benefitting from performance improvements of pipelining.
+
+## Load testing Dragonfly `c7gn.12xlarge`
+
+Next thing we tried running Dragonfly on the next generation instance with the same number of vCPUs (48).
+We used the same `c7gn.16xlarge` for running `memtier_benchmark` and we used the same commands
+to test writes, reads and pipelined reads:
+
+| Test          | Ops/sec   | Avg. Latency (us) | P99.9 Latency (us) |
+|---------------|-----------|-------------------|--------------------|
+| Write-Only    | 5.2M      | 250               | 631                |
+| Read-Only     | 6M        | 271               | 623                |
+| Pipelined Read| 8.9M      | 323               | 839                |
```

Comment:
Having a real table is much more readable, maybe use tables instead of console output above?

Reply:
I prefer to keep it for the first experiment to show the authenticity of the results.